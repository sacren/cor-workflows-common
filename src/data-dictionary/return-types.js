/* Copyright Â© 2016 Kuali, Inc. - All Rights Reserved
 * You may use and modify this code under the terms of the Kuali, Inc.
 * Pre-Release License Agreement. You may not distribute it.
 *
 * You should have received a copy of the Kuali, Inc. Pre-Release License
 * Agreement with this file. If not, please write to license@kuali.co.
 */
export const BOOLEAN = 'boolean'
export const CATEGORY = 'category'
export const CATEGORY_LIST = 'category-list'
export const FORM = 'form'
export const GROUP = 'group'
export const GROUP_LIST = 'group-list'
export const NUMBER = 'number'
export const ROLE = 'role'
export const SUBFLOW = 'subflow'
export const TEXT = 'text'
export const TEXT_LIST = 'text-list'
export const USER = 'user'
export const OPTIONS = 'options'
export const ALL = '*'
