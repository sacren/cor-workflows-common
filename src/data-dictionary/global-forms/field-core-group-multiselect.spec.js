/* Copyright Â© 2016 Kuali, Inc. - All Rights Reserved
 * You may use and modify this code under the terms of the Kuali, Inc.
 * Pre-Release License Agreement. You may not distribute it.
 *
 * You should have received a copy of the Kuali, Inc. Pre-Release License
 * Agreement with this file. If not, please write to license@kuali.co.
 */

import GroupMultiselect from './field-core-group-multiselect'
import { GROUP, GROUP_LIST, ROLE, TEXT, TEXT_LIST } from '../return-types'
import FieldTest from '../../test/utils/fields'

FieldTest(
  GroupMultiselect,
  [GROUP, GROUP_LIST, ROLE, TEXT, TEXT_LIST],
  [GROUP_LIST, TEXT_LIST]
)
