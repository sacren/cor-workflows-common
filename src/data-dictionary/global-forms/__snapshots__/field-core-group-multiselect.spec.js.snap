// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoreGroupMultiselect throw an error if you get children 1`] = `"received is not a function"`;
